### ISSUE TITLE
 Only variables should be passed by reference in ~/Code/Sites/purewater/web/app/mu-plugins/controller/src/Loader/Controller.php on line 46

### ISSUE THREAD

**Issue by christianmagill** [soberwp/controller#4]
I'm receiving the error with a basic test.

templates/page.blade.php

```
@extends('layouts.base')
@section('content')
  @while(have_posts()) @php(the_post())
  @include('partials.page-header')
  @include('partials.content-page')
  @endwhile
  {{$test}}
@endsection

```

controllers/page.php

```
<?php

namespace App;

use Sober\Controller\Controller;

class Page extends Controller
{
    /**
     * Protected methods will not be passed to the template
     */
    protected function hidden()
    {

    }

    /**
     * Return test data
     *
     * @return array
     */
    public function test()
    {
        return 'Testing';
    }
}
```

Everything else is setup via defaults.

**Comment by darrenjacoby** [comment]
@christianmagill please download 1.0.0-alpha4. That should resolve all issues at this stage.

**Comment by aaroneight** [comment]
I'm also getting this notice on alpha4

`Only variables should be passed by reference in /srv/www/viewfind.com/current/web/app/mu-plugins/controller/src/Loader.php on line 76`

**Comment by christianmagill** [comment]
I'm getting error on alpha4 as well line 76.

**Comment by darrenjacoby** [comment]
@aaroneight @christianmagill error should be resolved in 1.0.0-alpha5.

PHP function end() throws a warning if a var isn't provided. 

https://github.com/soberwp/controller/commit/f8220753ab05a3f493a721b0eabb43e8868f8af5#diff-3864ffe37f3a1de39876ea62cae525f3

Let me know if it resolves the issue.

**Comment by christianmagill** [comment]
That fixes the issue for me. Much thanks!
